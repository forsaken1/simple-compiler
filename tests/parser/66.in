float i, j, k, f, g;