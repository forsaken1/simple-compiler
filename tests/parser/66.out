(,)
|
+---(i)
|
+---(,)
    |
    +---(j)
    |
    +---(,)
        |
        +---(k)
        |
        +---(,)
            |
            +---(f)
            |
            +---(g)